frontend/www/js/omegaup/time.ts

```diff
@@ -155,6 +155,13 @@ export function formatTimestamp(date: Date): string {
   ).padStart(2, '0')}`;
 }
 
+export function formatTimestampSecondLess(date: Date): string {
```

Comment:
```suggestion
export function formatDateLocalHHMM(date: Date): string {
```

qué tal en vez de decir qué se excluye, decimos qué se incluye?